ui/components/multichain/network-manager/hooks/useAdditionalNetworkHandlers.ts

```diff
@@ -16,40 +11,12 @@ export const useAdditionalNetworkHandlers = () => {
   const handleAdditionalNetworkClick = useCallback(
     async (network: UpdateNetworkFields) => {
       await dispatch(hideModal());
-      const requestResult = (await dispatch(
-        requestUserApproval({
```

Comment:
Removed the extra "Want to add this..." modal when choosing from the pre-defined networks